tests/lean/type_error_at_eval_expr.lean.expected.out

```diff
@@ -1,6 +1,6 @@
 type_error_at_eval_expr.lean:3:0: error: eval_expr failed due to type error
 nested exception message:
-type mismatch at definition '_eval_expr.16.423', has type
+type mismatch at definition '_eval_expr.16.436', has type
   list ℕ
 but is expected to have type
   ℕ
```

Comment:
I think this is a good idea. BTW, we should have an option for disabling this kind of cleanup. The option is mostly for ourselves when debugging crazy bugs.